conf/messages.en

```diff
@@ -131,6 +131,10 @@ identification.checkIdentificationAnswers.heading = Check your answers
 identification.checkIdentificationAnswers.authorisations.subheading = Authorisations
 identification.checkIdentificationAnswers.addOrRemoveAuthorisations = Add or remove authorisations
 
+#location ogf goods
+
+locationOfGoods.caption = Arrivals - Location of goods
```

Comment:
Are we doing captions in Arrivals now? I thought it was just Departures

Reply:
It's in the content so we put it in, will see if it is to be added to all pages